lennykamande/stackoverflow.github.io
# Ft post a question 159850821

## What does this PR do?
Posts a question by user in the system.
### Description of Task to be completed?
Have the following UI available.
  * gets all questions asked and returns in json formart

### How should this be manually tested?
Make sure that you have installed Postman or ARC

* Clone the repo on to your local machine
* Cd into the folder 
* Run Python run.py 
* Open up Postman 
* Make a post request to http://localhost/5000/api/v1/questions with {"title":"Your quiz", "description": "explain your quiz", "user_id":"user_id"}

### Any background context you want to provide?
There are user interfaces for the stackoverflow Lite that have not been developed yet.
### What are the relevant pivotal tracker stories?
#159850821
